### ISSUE TITLE
Results generation on 3D KITTI dataset test set for server submission

### ISSUE THREAD

**Issue by IbrahimUWA** [LittlePey/SFD#64]
Hello author,
Recently, I have developed an object detection model and trained it on 3D KITTI dataset. Now, I would like to test on the test set and generate the results for benchmark online submission. I came across your work which is aligned with my work. I need your help if you provide me the right code for test results generation.

Best regards
Ibrahim
PhD student, University of Western Australia

**Comment by faziii0** [comment]
@IbrahimUWA 
Have you figured out. If you did can you help me in this regards

**Comment by Feidashen1** [comment]
> Hello author, Recently, I have developed an object detection model and trained it on 3D KITTI dataset. Now, I would like to test on the test set and generate the results for benchmark online submission. I came across your work which is aligned with my work. I need your help if you provide me the right code for test results generation.
> 
> Best regards Ibrahim PhD student, University of Western Australia

Hi, have you successfully submitted it to benchmark ？
I uploaded my work, but the result is much worse than on the validation set, much lower than SFD, but it is not like this on the validation set.
Or if I have some operational issues, could you give me some suggestions or discuss them together？
